extensions/package-vulnerability-scanner/src/components/ContentCard.vue

```diff
@@ -78,10 +78,10 @@ function handleClick() {
 
 <template>
   <div
-    class="border border-gray-300 hover:border-gray-400 rounded-lg p-4 shadow-sm hover:shadow-md transition-shadow cursor-pointer bg-white group"
+    class="min-w-0 border border-gray-300 hover:border-gray-400 rounded-lg p-4 shadow-sm hover:shadow-md transition-shadow cursor-pointer bg-white group"
     @click="handleClick"
   >
-    <div class="flex justify-between items-start mb-2">
+    <div class="flex justify-between items-start mb-2 gap-4">
```

Comment:
Here is how the truncation for long titles happens. `min-w-0` sets `min-width: 0` which changes the initial size of grid items from `min-width: auto` allowing the `truncate` class to work for the title.

`gap-4` provides a space between the title and the `ColorBadge` if the title is very long.